lib/value/src/kind/merge.rs

```diff
@@ -60,7 +30,16 @@ impl Kind {
         };
     }
 
+    /// Returns the union of self and other.
+    #[must_use]
+    pub fn union(&self, other: Self) -> Self {
+        let mut kind = self.clone();
+        kind.merge_keep(other, false);
+        kind
+    }
+
     /// Merge `other` into `self`, optionally overwriting on conflicts.
+    // deprecated
```

Comment:
Probably good to file an issue to remove the use of this method in a follow-up PR.